### ISSUE TITLE
Installing on linux via curseforge creates files with full paths as name

### ISSUE THREAD

**Issue by sam-k-taylor** [corporategoth/rotationmaster#14]
![image](https://user-images.githubusercontent.com/71709524/187615253-deaad9cb-78b7-499b-b3d5-866021605157.png)
other addons dont do this

**Comment by JemAs** [comment]
m1 mac appears to have a similar issue.

**Comment by fontivan** [comment]
I'm also seeing this, however if I manually download the zip from curseforge and unzip it myself it does not have the same issue

**Comment by sam-k-taylor** [comment]
yeah thats what I have done, not sure if this is an issue with rotation master or curseforge

**Comment by corporategoth** [comment]
I've seen reports of issues from mac users too.  But I'm not sure why.  I'm only using standard windows zip (via. powershell) to compress these archives.  For example:
```
Compress-Archive -Path RotationMaster -DestinationPath RotationMaster-$version.zip
```

Obviously this looks like it's on linux.  Both linux and mac are unix based, therefore use '/' for paths, not '\', so that may have something to do with it.  But if that was the case, *ALL* zip files would have this problem.

**Comment by captnrexx** [comment]
zip knows the Parameter -j (junk path). Maybe compress-archive has something  similar

**Comment by kriss-s** [comment]
Implementing github actions with BigWigs packager would solve the zip problem, as well as allow for people using other addon managers downloading directly from github.
See https://github.com/BigWigsMods/packager and https://github.com/BigWigsMods/packager/wiki/GitHub-Actions-workflow for a complete example. And you won't have to do the release packaging and uploading to different hosts manually

**Comment by corporategoth** [comment]
Thanks for the suggestion @kriss-s 

I just tried that - does someone want to see if 1.4.3 beta 2 fixes this issue?

**Comment by kriss-s** [comment]
> Thanks for the suggestion @kriss-s
> 
> I just tried that - does someone want to see if 1.4.3 beta 2 fixes this issue?

That fixed the file paths inside the archive, thank you! :)
However the addon manager i use didn't pick up the release for some reason, but it's quite a finicky one, it could perhaps be just because it's marked as pre-release. Not quite sure.

I'm using this addon manager https://github.com/ogri-la/strongbox

**Comment by corporategoth** [comment]
Fixed in 1.4.3
